### PR TITLE
feat(ui-kit): improve UI Kit components darkmode

### LINKED ISSUE #4069: Add darkmode support for the Loading data indicator

### LINKED ISSUE #4076: Add darkmode support: improve UI Kit components
- [x]  Input in darkmode when autofilled has a white background (maybe could be done in website too)

Image

- [x] Account selected
- [x] Panel bg

### PR DESCRIPTION
**Pull request by evavirseda**
# Description of change

Please write a summary of your changes and why you made them.

## Links to any relevant issues

fixes https://github.com/iotaledger/iota/issues/4076
fixes https://github.com/iotaledger/iota/issues/4069

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Bug fix (a non-breaking change which fixes an issue)
- Enhancement (a non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Documentation Fix

## How the change has been tested

Describe the tests that you ran to verify your changes.

Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by brancoder — APPROVED** [review]
LGTM :+1:

File: apps/ui-kit/src/lib/components/atoms/radio-button/RadioButton.tsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ function RadioButton({
                 />
                 <span
                     className="absolute
-                    text-neutral-40 peer-checked:text-primary-30 peer-disabled:opacity-40 peer-checked:peer-disabled:text-neutral-40 dark:text-neutral-60 dark:peer-checked:peer-disabled:text-neutral-40 [&_svg]:h-6 [&_svg]:w-6
+                    text-neutral-40 peer-checked:text-primary-30 peer-disabled:opacity-40 peer-checked:peer-disabled:text-neutral-40 dark:text-neutral-60 dark:peer-checked:text-primary-80 dark:peer-checked:peer-disabled:text-neutral-40 [&_svg]:h-6 [&_svg]:w-6
```
Comment (cpl121) [inline apps/ui-kit/src/lib/components/atoms/radio-button/RadioButton.tsx]: It doesn't look like the same color on the radio button

Captura desde 2024-11-25 09-01-33

image (21)

**Review by cpl121 — CHANGES_REQUESTED** [review]

File: apps/ui-kit/src/lib/styles/index.css [hunk 2]
```diff
@@ -45,6 +45,18 @@
...
+
+    input:-webkit-autofill,
+    input:-webkit-autofill:enabled,
+    input:-webkit-autofill:hover,
+    input:-webkit-autofill:focus {
+        appearance: none;
+        background-image: none !important;
+        background-color: transparent !important;
+        color: #545e6e !important;
+        transition: background-color 5000s ease-in-out 0s;
```
Comment (VmMad) [inline apps/ui-kit/src/lib/styles/index.css:55]: Why is the transition needed?

**Review by VmMad — CHANGES_REQUESTED** [review]
Overall looks good. There's one more thing that I just realized it's different: The input focus color. Do you mind changing it here too?
Left us, right figma.
image

File: apps/ui-kit/src/lib/styles/index.css (hunk 2 above)
Comment (marc2332) [reply apps/ui-kit/src/lib/styles/index.css:55]: 5s 💀

Comment (evavirseda) [reply apps/ui-kit/src/lib/styles/index.css:55]: the background-color transition prevents a flicker when browsers like Chrome apply autofill's default color (yellow or gray) before custom styles take effect and the duration (5000s) ensures the change is imperceptible

**Review by VmMad — COMMENTED** [review]
sweet 🍬

**Review by VmMad — APPROVED** [review]

**Review by cpl121 — APPROVED** [review]

**Pull request merged**